tox.ini

```diff
@@ -5,7 +5,8 @@ envlist=docs
 [doc8]
 # D001: line too long
 # D002: ignore trailing whitespace
-ignore = D001,D002
+# D004: carriage returns
+ignore = D001,D002,D004
```

Comment:
@PrecisionWordcraft: Is this a local only change?  Can this PR be closed?

Reply:
No, I created the PR so that it can be evaluated for adoption across repos.